src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/StateSynchronizationMenuItems.cs

```diff
@@ -7,13 +7,42 @@
 using System.IO;
 using System.Collections.Generic;
 using UnityEngine.SceneManagement;
+using System.Collections;
 
 namespace Microsoft.MixedReality.SpectatorView.Editor
 {
     public static class StateSynchronizationMenuItems
     {
         private static IEqualityComparer<IAssetCache> assetTypeComparer = new AssetCacheTypeEqualityComparer();
 
+        private const string disableSpectatorViewPreBuild = "Spectator View/Disable Spectator View Pre-build steps";
+        public static bool DisablePreBuildSteps
+        {
+            get
+            {
+                return PlayerPrefs.GetInt(disableSpectatorViewPreBuild, 0) > 0;
+            }
+            private set
+            {
+                PlayerPrefs.SetInt(disableSpectatorViewPreBuild, value ? 1 : 0);
+                PlayerPrefs.Save();
+            }
+        }
+
+        private const string disableUpdatingMenuItem = "Spectator View/Disable updating asset caches when building";
+        public static bool DisableUpdatingAssetCaches
+        {
+            get
+            {
+                return PlayerPrefs.GetInt(disableUpdatingMenuItem, 0) > 0;
+            }
+            private set
+            {
+                PlayerPrefs.SetInt(disableUpdatingMenuItem, value ? 1 : 0);
+                PlayerPrefs.Save();
+            }
+        }
```

Comment:
I would prefer if these were both stated in the positive (Enable Spectator View Pre-Build Steps, Enable Updating Asset Caches When Building) and then have them be true by default. I find combining checked state with negatively-termed options to be kind of confusing